compiler/rustc_expand/src/expand.rs

```diff
@@ -564,7 +564,7 @@ impl<'a, 'b> MacroExpander<'a, 'b> {
                             derive_invocations.reserve(derives.len());
                             derives
                                 .into_iter()
-                                .map(|DeriveResolution { path, item, exts: _, is_const }| {
+                                .map(|DeriveResolution { path, item, is_const, .. }| {
                                     // FIXME: Consider using the derive resolutions (`_exts`)
```

Comment:
The `SyntaxExtension` was preserved because of this FIXME.
Any fix will require preserving them again.

Reply:
I removed this commit and added a new one that adds a comment to `DeriveResolution` explaining this.